src/modules/ztsi/src/lib/Ztsi.cpp

```diff
@@ -290,20 +314,27 @@ unsigned int Ztsi::GetMaxPayloadSizeBytes()
 
 Ztsi::EnabledState Ztsi::GetEnabledState()
 {
-    AgentConfiguration configuration = {g_defaultServiceUrl, g_defaultEnabled};
+    AgentConfiguration configuration = {g_defaultEnabled, g_defaultMaxScheduledAttestationsPerDay , g_defaultMaxManualAttestationsPerDay};
     return (MMI_OK == ReadAgentConfiguration(configuration)) ? (configuration.enabled ? EnabledState::Enabled : EnabledState::Disabled) : EnabledState::Unknown;
 }
 
-std::string Ztsi::GetServiceUrl()
+int Ztsi::GetMaxScheduledAttestationsPerDay()
 {
-    AgentConfiguration configuration = {g_defaultServiceUrl, g_defaultEnabled};
-    return (MMI_OK == ReadAgentConfiguration(configuration)) ? configuration.serviceUrl : g_defaultServiceUrl;
+    AgentConfiguration configuration = {g_defaultEnabled, g_defaultMaxScheduledAttestationsPerDay , g_defaultMaxManualAttestationsPerDay};
+    // this is probably where things are going wrong
```

Comment:
>// this is probably where things are going wrong [](http://example.com/codeflow?start=4&length=48)

1. Start comments with uppercase ("This is...").

2. If the comment includes "probably" then the comment should be replaced by a code change to remove this uncertainty. Remove comment?